static/coffee/controllers.coffee

```diff
@@ -270,7 +270,18 @@ controllers.controller('MessagesController', ['$scope', '$timeout', '$uibModal',
     )
 
   $scope.fetchOldItems = (search, startTime, page) ->
-    return MessageService.fetchOld(search, startTime, page)
+    $scope.autoRefresh = ->
```

Comment:
This doesn't look like it can work with multiple pages - different pages will overwrite all of $scope.items